.github/workflows/alpha-integration-tests.yaml

```diff
@@ -0,0 +1,47 @@
+name: DataRepo Alpha Integration Tests
+on:
+  # Jenkins will kick this job off manually after Alpha is deployed
+  workflow_dispatch: {}
+jobs:
+  alpha-integration-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: "Checkout code"
+      uses: actions/checkout@v2
+    - name: "Import Vault secrets"
+      id: vault
+      uses: hashicorp/vault-action@c8b383ee
+      with:
+        url: ${{ secrets.VAULT_ADDR }}
+        method: approle
+        roleId: ${{ secrets.ALPHA_ROLE_ID }}
+        secretId: ${{ secrets.ALPHA_SECRET_ID }}
+        secrets: |
+          secret/dsde/datarepo/alpha/datarepo-api-sa key | B64_APPLICATION_CREDENTIALS ;
+    - name: "Write Vault tokens"
+      run: |
+        base64 --decode <<< ${{ steps.vault.outputs.B64_APPLICATION_CREDENTIALS }} > ${GOOGLE_APPLICATION_CREDENTIALS}
+        jq -r .private_key ${GOOGLE_APPLICATION_CREDENTIALS} > ${GOOGLE_SA_CERT}
+
+        # export environment variables
+        echo "::set-env name=GOOGLE_APPLICATION_CREDENTIALS::${GOOGLE_APPLICATION_CREDENTIALS}"
```

Comment:
```suggestion
        echo "::set-env name=GOOGLE_APPLICATION_CREDENTIALS::${GOOGLE_APPLICATION_CREDENTIALS}"
                ::add-mask::{GOOGLE_APPLICATION_CREDENTIALS}

```

Reply:
This won't prevent the contents of the file from being printed.